src/pybind/mgr/dashboard/controllers/iscsi.py

```diff
@@ -347,10 +347,10 @@ def set(self, target_iqn, new_target_iqn=None, target_controls=None, acl_enabled
                                      component='iscsi')
 
         settings = IscsiClient.instance(gateway_name=gateway).get_settings()
-        new_portal_names = {p['host'] for p in portals}
-        old_portal_names = set(config['targets'][target_iqn]['portals'].keys())
-        deleted_portal_names = list(old_portal_names - new_portal_names)
-        validate_rest_api(deleted_portal_names)
+        # new_portal_names = {p['host'] for p in portals}
+        # old_portal_names = set(config['targets'][target_iqn]['portals'].keys())
+        # deleted_portal_names = list(old_portal_names - new_portal_names)
+        # validate_rest_api(deleted_portal_names)
```

Comment:
IMO this check is useless. It also prevents a target from being updated with multiple portals, at least one of which is offline.